### PR TITLE
Dark mode for Dropdown, List and File Upload Drop Zone

### LINKED ISSUE #802: Dark Mode - Dropdown, File upload dropzone, List

### PR DESCRIPTION
**Pull request by msankaran0712**
Fixes #802 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by coliff — COMMENTED** [review]
SUPPORT.md isn't needed in the stencil-workspace folder anymore, can you please get latest from main branch? (it'll build without failure without a SUPPORT.md in the folder with latest from main)

File: stencil-workspace/src/components/modus-accordion-item/modus-accordion-item.scss [hunk 1]
```diff
@@ -1,9 +1,13 @@
+@import './modus-accordion-item.vars';
+
 .accordion-item {
-  border-bottom: $rem-1px solid $col_gray_1;
+  border-bottom: $rem-1px solid $modus-accordion-border-color;
   display: flex;
   flex-direction: column;
   font-family: $primary-font;
   position: relative;
+  background-color: $modus-accordion-bg;
```
Comment (coliff) [inline stencil-workspace/src/components/modus-accordion-item/modus-accordion-item.scss:9]: these should be alphabetized ideally...

Comment (msankaran0712) [reply stencil-workspace/src/components/modus-accordion-item/modus-accordion-item.scss:9]: @coliff I created a separate [PR](https://github.com/trimble-oss/modus-web-components/pull/850/files#diff-437bfb1ec2bb02b4aabc05f9b5bfee09b3be38e5b4e7beec28629dc5ebcf7ac2) for fixing this comment.

**Comment by coliff** [comment]
Hey - the color for the dropdown button in dark mode (and any other Primary buttons) should be `#019aeb` 

See these pages in dark-mode for examples:
https://modus.trimble.com/components/web/buttons/#structural-progression
https://modus.trimble.com/components/web/dropdowns/#specifications

**Comment by msankaran0712** [comment]
> Hey - the color for the dropdown button in dark mode (and any other Primary buttons) should be `#019aeb`
> 
> See these pages in dark-mode for examples: https://modus.trimble.com/components/web/buttons/#structural-progression https://modus.trimble.com/components/web/dropdowns/#specifications

Please check now

**Review by coliff — APPROVED** [review]

**Pull request merged**